dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-http.ts

```diff
@@ -192,7 +192,7 @@ export function useHttp(model: { [field: string]: any }): IJsonItem[] {
     ...useCustomParams({
       model,
       field: 'localParams',
-      isSimple: true
+      isSimple: false
```

Comment:
Are you sure you need the direct and the type here?

Reply:
+1

Reply:
Yes, because the front end displays drop-down selections for in/out and VARCHAR